src/chessboard/components/Board.tsx

```diff
@@ -33,32 +34,47 @@ export function Board() {
   }, []);
 
   return boardWidth ? (
-    <div ref={boardRef} style={{ position: "relative" }}>
-      <Squares />
-      <Arrows />
+    <div style={{ perspective: "1000px" }}>
```

Comment:
I've not used this property before, how does this being here not affect every board?

Reply:
CSS perspective property does not affect 2D elements. The perspective property is used in conjunction with 3D transformations in CSS to create a 3D perspective effect. It doesn't have any impact on purely 2D elements or transformations.